### PR TITLE
Split PackageManager's Program.cs in multiple files

### PR DESCRIPTION
**Pull request by victorhurdugaci**
Split PackageManager's program.cs in multiple files = No longer a mess

Didn't make any other code changes other that updating where the parameters are used.

cc @ChengTian @davidfowl

### CODE REVIEW

File: src/Microsoft.Framework.PackageManager/ConsoleCommands/PackagesConsoleCommand.cs [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                    {
+                        // Implicitly commit changes before pull
+                        var commitOptions = new CommitOptions
+                        {
+                            Reports = reports,
+                            SourcePackages = argSource.Value
+                        };
+                        var commitCommand = new CommitCommand(commitOptions);
+                        success = commitCommand.Execute();
+                        if (!success)
```
Comment (BrennanConroy) [inline src/Microsoft.Framework.PackageManager/ConsoleCommands/PackagesConsoleCommand.cs]: Use ```return success ? 0 : 1;``` like everywhere else, and move the creation of the variable to right above this line

**Comment by BrennanConroy** [comment]
Looks pretty nice.

File: src/Microsoft.Framework.PackageManager/ConsoleCommands/PackConsoleCommand.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+using System;
+using System.IO;
+using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Runtime.Common.CommandLine;
+
+namespace Microsoft.Framework.PackageManager
+{
+    internal class PackConsoleCommand
+    {
+        public static void Register(CommandLineApplication cmdApp, ReportsFactory reportsFactory,IServiceProvider serviceProvider)
```
Comment (BrennanConroy) [inline src/Microsoft.Framework.PackageManager/ConsoleCommands/PackConsoleCommand.cs]: Add space after comma

Comment (victorhurdugaci) [inline src/Microsoft.Framework.PackageManager/ConsoleCommands/PackConsoleCommand.cs]: Fixed

**Comment by ChengTian** [comment]
Looks good to me :shipit:

**Pull request merged**